consensus/src/consensus.rs

```diff
@@ -153,6 +154,11 @@ impl<R: ConsensusRpc> ConsensusClient<R> {
             .await
             .map_err(|_| eyre!("could not fetch bootstrap"))?;
 
+        let is_valid = self.is_valid_blockhash(bootstrap.header.slot);
+        if !is_valid {
+            return Err(ConsensusError::InvalidTimestamp.into());
```

Comment:
Can we add a new error called `CheckpointToOld` instead so we can provide better error messages?